### ISSUE TITLE
`fn` layer not being switched to in macos kanata config

### ISSUE THREAD

**Issue by kd-labs** [dreamsofcode-io/home-row-mods#8]
https://github.com/dreamsofcode-io/home-row-mods/blob/932ef940240b8831eddaadbd8fa138730eb7e5ea/kanata/macos/kanata.kdb#L31

In the mac os kanata config, I am not able to understand the purpose of defining the `fn` layer when it is not being switched, and thus not being used.

Could someone plz correct my understanding.

**Comment by elliottminns** [comment]
You're correct! I must have forgotten to add in the fn layer switch. Will send up a change now.
